### PR TITLE
Catch testthat test failures in make

### PR DESCRIPTION
**Pull request by ashiklom**
Before, testthat errors were reported, but didn't cause `make test` to actually fail because the command itself still completed successfully.
Setting `reporter = "stop"` means that failing tests are treated as errors.

Also, running `devtools::document()` on all packages with the latest version of Roxygen made minor edits to a lot of documentation (e.g. removing trailing empty lines, updating roxygen version in DESCRIPTION files), which are also included here.

### CODE REVIEW

File: modules/emulator/NAMESPACE [hunk 1]
```diff
@@ -4,6 +4,8 @@ S3method(calcSpatialCov,list)
 S3method(calcSpatialCov,matrix)
 S3method(p,jump)
 S3method(p,mvjump)
+S3method(plot,jump)
```
Comment (robkooper) [inline modules/emulator/NAMESPACE]: is this correct?

Comment (ashiklom) [inline modules/emulator/NAMESPACE]: That's a question for @mdietze , it's his function. It _looks_ like `jump` is an S4 class, so I don't think that's right. I changed it in cbdf8b0, but I can revert if necessary.

**Comment by istfer** [comment]
sorry, I wasn't taking good care of this package, these functions are currently not being used, feel free to modify as you see fit

**Comment by mdietze** [comment]
@ashiklom this was passing but just failed again after #1335

**Comment by ashiklom** [comment]
@mdietze It's failing because #1335 added failing tests, to which Travis is now sensitive. So blame @infotroph , don't shoot the messenger :wink: 

https://travis-ci.org/PecanProject/pecan/builds/221486312#L2987

**Comment by ashiklom** [comment]
@mdietze It's failing on a `data.atmosphere` test. See [line 2987 of the build log](https://travis-ci.org/PecanProject/pecan/builds/221486312#L2987).

**Pull request merged**